css-four-column-layout/styles.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  max-width: 1200px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-one-fourth {
+  flex-basis: 25%;
+}
+
+.screens {
+  font-family: Barlow, sans-serif;
+  background-color: #f7f8fa;
+  text-align: center;
+}
+
+.screen {
+  width: 100%;
+}
+
+.column-one-fourth > .icon {
+  width: 25%;
+}
+
+.padding {
+  padding-left: 2rem;
+}
+
+h3 {
+  font-size: 0.8rem;
+  margin-bottom: 1.5rem;
+}
+
+p {
+  font-size: 0.8rem;
+  line-height: 0.5rem;
+}
+
+h2 {
+  font-size: 1.2rem;
+}
+
+.column-full {
+  flex-basis: 100%;
+  justify-content: center;
+  text-align: center;
```

Comment:
You should keep these on a separate class

Reply:
<img width="901" alt="Screen Shot 2022-07-09 at 2 43 57 PM" src="https://user-images.githubusercontent.com/97363006/178123651-70641aa8-c17e-4509-8fa9-656d8829d03e.png">